support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -120,49 +143,59 @@ export const productCatalogDescription: Record<string, ProductDescription> = {
 		ratePlans: {
 			Monthly: {
 				billingPeriod: 'Monthly',
+				deliverableTo: gwDeliverableCountries,
 			},
 			OneYearGift: {
 				billingPeriod: 'Annual',
+				deliverableTo: gwDeliverableCountries,
 			},
 			Annual: {
 				billingPeriod: 'Annual',
+				deliverableTo: gwDeliverableCountries,
 			},
 			SixWeekly: {
 				billingPeriod: 'Monthly',
+				deliverableTo: gwDeliverableCountries,
 			},
 			Quarterly: {
 				billingPeriod: 'Quarterly',
+				deliverableTo: gwDeliverableCountries,
 			},
 			ThreeMonthGift: {
 				billingPeriod: 'Monthly',
+				deliverableTo: gwDeliverableCountries,
 			},
 		},
-		deliverableTo: gwDeliverableCountries,
 	},
 	GuardianWeeklyDomestic: {
 		label: 'The Guardian Weekly',
 		benefits: [],
 		ratePlans: {
 			Monthly: {
 				billingPeriod: 'Monthly',
+				deliverableTo: gwDeliverableCountries,
```

Comment:
Does this country list need to be split between  `GuardianWeeklyDomestic` & `GuardianWeeklyRestOfWorld`?